### PR TITLE
Feature/test requirements

### PR DESCRIPTION
**Pull request by skovbakke**
Added a section explaining our use of tests in proxy Helm charts.

### CODE REVIEW

File: helm/test.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Testing Services
+
+All of the services develloped for the platform should pass basic acceptance tests before they are deployed. 
+The tests should verify the core functionality of the service. The tests should catch configuration issues or problems with the service and ensure that the service functions as is inteded. 
+
+Generally the tests are located in a Helm proxy chart for the service. The tests are located in a folder called `test`. Before any merge of changes onto the main branch, a github workflow automatically runs the test in a `kind` cluster.
```
Comment (jacobcrawford) [inline helm/test.md:6]: For consistency, I think we should use the helm test hooks for acceptance tests.  
Then a test is described in yaml as a Pod that runs a test script, and we can use the `helm test xxx` command as a test engine. That way we can run tests inside the cluster and we are not dependent on the host system having packages installed.
See https://github.com/distributed-technologies/services-kafka/blob/main/.github/workflows/test_create.yaml
for reference.

File: helm/test.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Testing Services
+
+All of the services develloped for the platform should pass basic acceptance tests before they are deployed.
```
Comment (jaxels10) [inline helm/test.md:3]:
```suggestion
All of the services developed for the platform should pass basic acceptance tests before they are deployed. 
```

File: helm/test.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Testing Services
+
+All of the services develloped for the platform should pass basic acceptance tests before they are deployed. 
+The tests should verify the core functionality of the service. The tests should catch configuration issues or problems with the service and ensure that the service functions as is inteded.
```
Comment (jaxels10) [inline helm/test.md:4]:
```suggestion
The tests should verify the core functionality of the service. The tests should catch configuration issues or problems with the service and ensure that the service functions as intended. 
```

**Review by jaxels10 — CHANGES_REQUESTED** [review]

File: helm/test.md (hunk 1 above)
Comment (skovbakke) [reply helm/test.md:6]:
```suggestion
Generally the tests are located in a Helm proxy chart for the service. The tests are located in a folder called `test`. Before any merge of changes onto the main branch, a github workflow automatically runs the test in a `kind` cluster. 
See https://github.com/distributed-technologies/services-kafka/blob/main/.github/workflows/test_create.yaml
for reference.
```

**Review by jaxels10 — APPROVED** [review]
Looks good.

**Review by MartinSchmidt — APPROVED** [review]

**Pull request merged**